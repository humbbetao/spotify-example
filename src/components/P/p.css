p {
  margin: 0;
  border: 0;
  padding: 8px 0;
}
