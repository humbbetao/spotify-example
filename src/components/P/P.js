import React from 'react'
import './p.css'
export default function P({ children }) {
  return <p>{children}</p>
}
