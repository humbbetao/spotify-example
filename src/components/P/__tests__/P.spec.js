import React from 'react'
import { mount } from 'enzyme'
import P from '../P'

describe('Label', () => {
  it('should render correctly when children is not passed', () => {
    const wrapper = mount(<P />)
    expect(wrapper.children.length).toEqual(1)
    expect(wrapper).toMatchSnapshot()
  })

  it('should render correctly when children is passed', () => {
    const wrapper = mount(<P>Teste</P>)

    expect(wrapper).toMatchSnapshot()
    expect(wrapper.children.length).toEqual(1)
  })
})
