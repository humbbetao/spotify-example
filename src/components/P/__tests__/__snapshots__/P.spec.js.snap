// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label should render correctly when children is not passed 1`] = `ReactWrapper {}`;

exports[`Label should render correctly when children is passed 1`] = `ReactWrapper {}`;
